cli/autostart_test.go

```diff
@@ -158,4 +159,26 @@ func TestAutostart(t *testing.T) {
 		require.NoError(t, err, "fetch updated workspace")
 		require.Equal(t, expectedSchedule, *updated.AutostartSchedule, "expected default autostart schedule")
 	})
+
+	t.Run("BelowTemplateConstraint", func(t *testing.T) {
```

Comment:
**Question(gut-check)**: Should there be a similar case for _AboveTemplateConstraint_ ?

Reply:
I don't see any reason to enforce an upper bound on the interval between successive autostarts.